### ISSUE TITLE
Preserve data sparsity in the StandardScaler()

### ISSUE THREAD

**Issue by kapiblue** [JankowskiDaniel/Neural-Deep-Retina#28]
Preserve sparsity of the input data by using mean = 0.

This can be achieved by:

```python
from sklearn.preprocessing import StandardScaler

scaler = StandardScaler(with_mean=False)
scaler.fit(data)
```

Examine the performance

https://stackoverflow.com/questions/21875518/features-scaling-and-mean-normalization-in-a-sparse-matrix
